scripts/linters/python_linter_test.py

```diff
@@ -115,24 +115,6 @@ def test_jobs_are_listed_in_job_registry_file_with_duplicate_prod_job(self):
         self.assertEqual('Job registry', lint_task_report.name)
         self.assertTrue(lint_task_report.failed)
 
-    def test_custom_linter_with_test_only_in_non_test_file(self):
-        lint_task_report = python_linter.PythonLintChecksManager(
-            [INVALID_TEST_ONLY_FILEPATH], FILE_CACHE).check_non_test_files()
```

Comment:
Remove the `INVALID_TEST_ONLY_FILEPATH` file from the codebase.

Reply:
Done now.